src/cljs/main/broadfcui/main.cljs

```diff
@@ -112,6 +112,33 @@
                   "http://status.firecloud.org/"]
                  " for more information."])}))
 
+(defn status-alert [title message link]
+  [:div {:style {:borderBottom "1px solid" :borderBottomColor (:line-default style/colors)
+                 :backgroundColor (:exception-state style/colors) :padding "1rem"}}
+   [:div {:style {:color "#fff" :marginBottom "0.5rem" :fontWeight 600}}
+    [icons/ExceptionIcon {:size 18 :paddingRight "0.5rem"}] (if title title "Service Alert")]
```

Comment:
I want the padding to be between the icon and the title text, wouldn't that just put the padding on that entire unit?